src/Extension/Tasks/CodesnifferTask.yaml

```diff
@@ -26,6 +26,13 @@ placeholders:
       description: "Reports directory"
       type: string
       settingPaths: ["report_dir"]
+  - name: "%severity%"
+    value_resolver: OPTION
+    optional: true
+    configuration:
+      name: 'severity'
+      type: 'option'
```

Comment:
It seems to me that the `type` option changed its meaning from 'datatype' to `behavior`. Is it acceptable or can we perhaps use another configuration for that purpose?

Reply:
Fixed